nucleus/deployment/admin/src/main/java/org/glassfish/deployment/admin/AddLibraryCommand.java

```diff
@@ -117,17 +135,31 @@ public void execute(AdminCommandContext context) {
             List<UnprocessedChangeEvent> unprocessed = 
                 new ArrayList<UnprocessedChangeEvent>();
 
-            StringBuffer msg = new StringBuffer();
+            StringBuilder msg = new StringBuilder();
+            
+            ClassLoader commonLoader = commonClsLdr.getCommonClassLoader();
+            CurrentBeforeParentClassLoader loader = null;
+            if (commonLoader instanceof CurrentBeforeParentClassLoader){
+                loader = (CurrentBeforeParentClassLoader) commonLoader;
+            }
 
             for (File libraryFile : files) {
                 if (libraryFile.exists()) {
-                    DeploymentCommandUtils.renameUploadedFileOrCopyInPlaceFile(
+                    logger.log(Level.SEVERE, "ready to add new library");
```

Comment:
This should be INFO if this is meant for consumption by an administrator or WARNING at most. The Log message also isn't explanatory in any way as to what is required to fix the condition you are logging.